library/src/main/java/io/karn/notify/internal/NotificationInterop.kt

```diff
@@ -146,6 +146,11 @@ internal object NotificationInterop {
                 // The duration of time after which the notification is automatically dismissed.
                 .setTimeoutAfter(payload.meta.timeout)
 
+        if(payload.progress.showProgress){
+            if (payload.progress.showIndeterminate) builder.setProgress(0,0,true)
+            else builder.setProgress(100,payload.progress.determinateProgress,false)
+        }
+
```

Comment:
Nice, I like that you've defaulted to percentages! Just a minor comment about the formatting -- let's add spaces after the `if` and before the ending curly brace. and spaces after the commas when providing function params. Something like:

```kotlin
if (payload.progress.showProgress) {
    if (payload.progress.showIndeterminate) builder.setProgress(0, 0, true)
    else builder.setProgress(100, payload.progress.determinateProgress, false)
}
```

Reply:
ready boss